tsconfig.json

```diff
@@ -20,5 +20,6 @@
       "test-log": ["./test/lib/test-log"]
     }
   },
-  "include": ["test/**/*.test.ts", "test/**/*.test.tsx", "test/lib/**/*.ts"]
+  "include": ["test/**/*.test.ts", "test/**/*.test.tsx", "test/lib/**/*.ts"],
+  "exclude": ["node_modules"]
```

Comment:
Restored `"exclude": ["node_modules"]` removed from #66461

### Why?

`typescript@5.5.2` and `web-vitals@latest` have conflict with modifiers on `LargestContentfulPaint`

TS has `readonly` whereas `web-vitals` don't, causing type conflict:

```
error TS2687: All declarations of 'element' must have identical modifiers.
error TS2687: All declarations of 'id' must have identical modifiers.
...
```

We can simply ignore this by restoring `exclude node_modules`.

Reply:
I removed `"exclude": ["node_modules"]` in #66461 because it was not needed as it had no effect. As per the [docs](https://www.typescriptlang.org/tsconfig/#exclude):

> `exclude` only changes which files are included as a result of the [`include`](https://www.typescriptlang.org/tsconfig/#include) setting. A file specified by `exclude` can still become part of your codebase due to an `import` statement in your code, a `types` inclusion, a `/// <reference` directive, or being specified in the [`files`](https://www.typescriptlang.org/tsconfig/#files) list.
>
> It is not a mechanism that **prevents** a file from being included in the codebase - it simply changes what the [`include`](https://www.typescriptlang.org/tsconfig/#include) setting finds.

You can verify that by running `pnpm typescript --listFiles | wc -l` with and without it.

Adding it here again does not change the compiler errors you mentioned, as far as I can see. You could only suppress them with `skipLibCheck`, which I hope we won't need to do, and can find a better solution instead. Maybe it should be raised in https://github.com/GoogleChrome/web-vitals that https://github.com/GoogleChrome/web-vitals/blob/main/src/types.ts now has conflicting types with `typescript@5.5.2/node_modules/typescript/lib/lib.dom.d.ts`.

Reply:
Totally makes sense, removed it. Thank you!